packages/wrangler/src/__tests__/secret.test.ts

```diff
@@ -95,6 +96,83 @@ describe("wrangler secret", () => {
 				expect(std.err).toMatchInlineSnapshot(`""`);
 			});
 
+			it("should create secret:bulk", async () => {
+				writeFileSync(
+					"secret.json",
+					JSON.stringify({
+						"secret-name-1": "secret_text",
+						"secret-name-2": "secret_text",
+					})
+				);
+
+				// User counter to pass different secrets to the request mock
+				const counter = 0;
```

Comment:
This should be `let` not `const`

Reply:
You know to make it more clear I can make it let and use counter++ which should do the same thing.